app/javascript/components/profile/TestimonialsList.tsx

```diff
@@ -60,15 +68,13 @@ export default function TestimonialsList({
           <>
             <div className="testimonials">
               {resolvedData.results.map((t) => {
-                return (
-                  <Testimonial
-                    testimonial={t}
-                    open={t.uuid === selected}
-                    onClick={handleTestimonialOpen(t.uuid)}
-                    onClose={handleTestimonialClose}
-                    key={t.uuid}
-                  />
-                )
+                return cloneElement(children, {
```

Comment:
What was wrong with the old way of rendering things?

Reply:
Answered on Slack.